app/models/atmosphere/action.rb

```diff
@@ -0,0 +1,13 @@
+module Atmosphere
+  class Action < ActiveRecord::Base
```

Comment:
In the feature `Action` will also be connected with ApplianceType - it will be added in another PR - am I right?